pydrive2/test/README.rst

```diff
@@ -1,13 +1,30 @@
-Instructions
-------------
-
--  Install timeout-decorator and futures (used for parallel upload and
-download testing).
--  Create *configs* and *credentials* folder in working directory.
--  Put *client_secrets.json* file from API console into *configs* folder.
--  Put *clients_secrets_local.json* from API console, for native device in
-*configs* folder.
--  Put different *a.png* and *b.png* file in working directory.
--  Replace {{ }} sections in *settings/test2.yaml* with the relevant sections
-from your config file.
--  For ServiceAuth test, place PKCS12 file in working directory.
+Run tests locally
+-----------------
+
+1. Setup a Google service account for your Google Cloud Project:
+    - Sign into the `Google API Console <https://console.developers.google.com>`_
+    - Select or `Create a new <https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project>`_ project.
+    - `Enable the Drive API <https://developers.google.com/drive/api/v2/about-sdk>`_ from the **APIs & Services** **Dashboard** (left sidebar), click on **+ ENABLE APIS AND SERVICES**. Find and select the "Google Drive API" in the API Library, and click on the **ENABLE** button.
+    - Go back to **IAM & Admin** in the left sidebar, and select **Service Accounts**. Click **+ CREATE SERVICE ACCOUNT**, on the next screen, enter **Service account name** e.g. "PyDrive tests", and click **Create**. Select **Continue** at the next **Service account permissions** page, click at **+ CREATE KEY**, select **P12** and **Create**. Save generated **.p12** key file at your local disk.
+
+2. Edit `PyDrive default.yaml </pydrive2/test/settings/default.yaml>`_ by replacing **your-service-account-email** with your new created service account email and **your-file-path.p12** with path to the downloaded **.p12** key file.
+
+3.
+
+::
+
+    py.test -v -s -m "not manual"
```

Comment:
why do I need to specify some modifier in both cases? It feels that default should exclude some tests or include all of them?

Reply:
what happens if I run it w/o `-m`?

Reply:
do we need `-v -s`?

Reply:
All params `-v -s -m "not manual"` are not mandatory.
`-v -s` are used to run tests on Travis. I'm finding `-v` verbose option worth to use. `-s` seems also disabled redirection of tests output and we can see more output info in console.

it might be simpler to NOT divide doc for automated and manual tests sections(to not use -m "not manual" or "manual"). pytest will run all tests in this case.

Reply:
yep, sounds reasonable